mix.exs

```diff
@@ -69,9 +69,10 @@ defmodule Swarm.Mixfile do
   end
 
   defp aliases() do
-    case System.get_env("SWARM_TEST_DEBUG") do
-      "true" -> [test: "test --no-start --trace"]
-      _ -> [test: "test --no-start"]
+    if System.get_env("SWARM_TEST_DEBUG") do
```

Comment:
Hmm, if someone passes `SWARM_TEST_DEBUG=false` then this still will trigger tracing. Not really a big issue though.

Reply:
Yeah, I considered that too, in the end I decided it didn't make sense to ever set it to `false` since that's the default behavior. If it turns out that's a thing that is needed, we can always change it later.

Reply:
Yes, agreed